repology/parser/slackware.py

```diff
@@ -0,0 +1,96 @@
+# Copyright (C) 2016 Dmitry Marakasov <amdmi3@amdmi3.ru>
+# Copyright (C) 2017 David Spencer <idlemoor@slackbuilds.org>
+#
+# This file is part of repology
+#
+# repology is free software: you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation, either version 3 of the License, or
+# (at your option) any later version.
+#
+# repology is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with repology.  If not, see <http://www.gnu.org/licenses/>.
+
+import os
+import re
+import sys
+
+from repology.package import Package
+
+
+class SlackwareParser():
+    def __init__(self):
+        pass
+
+    def Parse(self, path):
+        packages = {}
+
+        # For all the Slackware packages:
+        maintainers = ['volkerdi@slackware.com']
```

Comment:
Hmm. Not sure about it, but let it be, since sometime we'll generate default maintainer anyway.